packages/expo-localization/android/src/main/java/expo/modules/localization/LocalizationUtils.kt

```diff
@@ -0,0 +1,40 @@
+package expo.modules.localization
+
+import android.text.TextUtils
+import java.lang.Exception
+import java.util.*
+import kotlin.collections.ArrayList
+
+val USES_IMPERIAL = listOf("US", "LR", "MM")
+
+val iSOCurrencyCodes: ArrayList<String> by lazy {
+  Currency.getAvailableCurrencies().map { it.currencyCode } as ArrayList<String>
+}
+
+fun getLocaleNames(locales: ArrayList<Locale>?) = locales?.map { it.toLanguageTag() } as ArrayList<String>
```

Comment:
```suggestion
fun getLocaleNames(locales: List<Locale>?) = locales?.map { it.toLanguageTag() } as ArrayList<String>
```
Why is the `as ArrayList` cast needed?

Reply:
We put the result of this function to `Bundle` with `putStringArrayList` so in the end we still need `ArrayList` not `List`.